src/modules/dex/directives/dexWatchlist/watchlist.html

```diff
@@ -25,7 +25,7 @@
                 <w-dex-exchange base-asset-id="$ctrl.baseAssetId" asset-id="{{::asset.id}}"></w-dex-exchange>
             </w-cell>
             <w-cell>
-                <w-change24 asset-from="$ctrl.baseAssetId"
+                <w-change24 direction-by-pair="::true" asset-from="$ctrl.baseAssetId"
```

Comment:
In case we prefer placing all attributes either all on one line or every on separate lines line break might be missing here.